examples/repl.rs

```diff
@@ -207,37 +230,13 @@ mod example {
                 ))?,
             },
             Some("bt") | Some("backtrace") => {
-                context.load_debuginfo_if_necessary()?;
-
-                let regs = context.remote()?.read_regs()?;
-
-                // Read stack
-                let mut stack: [usize; 1024] = [0; 1024];
-                unsafe {
-                    context
-                        .remote()?
-                        .read()
-                        .read(&mut stack, regs.rsp as usize)
-                        .apply()?;
-                }
-
-                let call_stack: Vec<_> = match parts.next() {
-                    Some("fp") | None => headcrab::symbol::unwind::frame_pointer_unwinder(
-                        context.debuginfo(),
-                        &stack[..],
-                        regs.rip as usize,
-                        regs.rsp as usize,
-                        regs.rbp as usize,
-                    )
-                    .collect(),
-                    Some("naive") => headcrab::symbol::unwind::naive_unwinder(
-                        context.debuginfo(),
-                        &stack[..],
-                        regs.rip as usize,
-                    )
-                    .collect(),
+                let bt_type = match parts.next() {
+                    Some("fp") | None => BtType::FramePtr,
+                    Some("naive") => BtType::Naive,
```

Comment:
Maybe move this code to `impl BtType`? Then we can have it all in one place - e.g.:
```rust
impl BtType {
    fn from_str(value: Option<&str>) {
        match value {
            Some("fp") | None => BtType::FramePtr,
            Some("naive") => BtType::default(),
        }
    }
}
```

Reply:
This can be done after #106 is merged. `HighlightAndComplete` has a `from_str` method. Implementing it would also give autocomplete and syntax highlighting.

Reply:
So I would just leave a comment for now here and make the refactoring after #106 is merged ?

Reply:
#106 was merged 4 hours ago.

Reply:
:)

Just saw it. So I will get on with it